qiskit/providers/models/backendstatus.py

```diff
@@ -66,3 +74,16 @@ def __eq__(self, other):
             if self.__dict__ == other.__dict__:
                 return True
         return False
+
+    def _repr_html_(self) -> str:
+        """Return html representation of the object"""
```

Comment:
Just a small nit, I would probably put something here about this only being used in ipython and/or jupyter. But not worth blocking over if everything else looks good.

Reply:
Done